### ISSUE TITLE
[Docs][MAUI Embedding] Update code snippet to use Uno.Sdk

### ISSUE THREAD

**Issue by kazo0** [unoplatform/uno.extensions#2329]
![image](https://github.com/unoplatform/uno.extensions/assets/4793020/a388355e-3ec9-46a3-a20d-c20ec8966481)

The code snippet for the "project file for the Uno class library" shows the old Microsoft.NET.Sdk usage

**Comment by agneszitte** [comment]
@kazo0 yes the entire page will need to be reviewed in terms of doc for the overview with the latest changes:
https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Maui/MauiOverview.html?tabs=single-project
